core/cog-launcher.c

```diff
@@ -69,6 +70,25 @@ on_action_open (G_GNUC_UNUSED GAction *action,
                               g_variant_get_string (param, NULL));
 }
 
+static gboolean
+decidePermissionRequest (WebKitWebView *webView, WebKitPermissionRequest *request, CogLauncher *launcher)
+{
+    if (!g_strcmp0 (launcher->permissions, "all"))
```

Comment:
Making this a boolean or enum as suggested above would avoid having to compare strings here.